include/mbgl/util/image.hpp

```diff
@@ -26,6 +28,18 @@ class Image {
           height(h),
           data(std::move(data_)) {}
 
+    Image(Image&& o)
+        : width(o.width),
+          height(o.height),
+          data(std::move(o.data)) {}
+
+    Image& operator=(Image&& o) {
```

Comment:
It was already non-copyable. And the added move constructor and move assignment operator are what the compiler would provide automatically anyway. (You should almost never write an explicit move constructor or move assignment operator.)